mongodb/src/main/java/org/apache/zeppelin/mongodb/MongoDbInterpreter.java

```diff
@@ -73,7 +73,15 @@ public void open() {
     }
     commandTimeout = Long.parseLong(getProperty("mongo.shell.command.timeout"));
     maxConcurrency = Integer.parseInt(getProperty("mongo.interpreter.concurrency.max"));
-    dbAddress = getProperty("mongo.server.host") + ":" + getProperty("mongo.server.port");
+
+		/* adding support for protocal like mongodb+srv for atlas cluster*/
+    String mongoProtocol = getProperty("mongo.server.protocol", "mongodb");
+		if ("mongodb".equalsIgnoreCase(mongoProtocol)){
```

Comment:
Could you please check your indentation settings?

Reply:
Sure will update indentation.